android/app/src/main/java/com/on/turip/ui/search/detail/SearchDetailActivity.kt

```diff
@@ -27,9 +49,89 @@ class SearchDetailActivity : BaseActivity<SearchDetailViewModel, ActivitySearchD
             this,
             object : OnBackPressedCallback(true) {
                 override fun handleOnBackPressed() {
-                    finish()
+                    if (turipWebChromeClient.isFullScreen()) {
+                        turipWebChromeClient.onHideCustomView()
+                    } else if (webView.canGoBack()) {
+                        webView.goBack()
+                    } else {
+                        finish()
+                    }
                 }
             },
         )
     }
+
+    private fun setupBindings() {
+        webView = binding.wvSearchDetailVideo
+    }
+
+    private fun setupWebView() {
+        turipWebChromeClient =
+            TuripWebChromeClient(
+                fullScreenView = binding.flSearchDetailVideoFullscreen,
+                onEnterFullScreen = ::enableFullscreen,
+                onExitFullScreen = ::disableFullscreen,
+            )
+
+        webView.applyVideoSettings()
+        webView.webChromeClient = turipWebChromeClient
+        webView.webViewClient = TuripWebViewClient(binding.pbSearchDetailVideo)
+        webView.addJavascriptInterface(
+            // TODO 생성자에 url의 일부 추출 필요, WebViewUtils에 구현한 String.extractVideoId() 사용 예정
+            WebViewVideoBridge("") { showWebViewErrorView() },
+            BRIDGE_NAME_IN_JS_FILE,
+        )
+        webView.loadUrl(LOAD_URL_FILE_PATH)
+    }
+
+    private fun enableFullscreen() {
```

Comment:
c: 
```suggestion
    private fun enableFullScreen() {
```

풀 스크린 ,,  , ?